test/verify/machineslib.py

```diff
@@ -22,6 +22,7 @@
 import subprocess
 import time
 import unittest
+import xml.etree.ElementTree as ET
```

Comment:
I can use sed instead of ET as well. I don't really have a preference here.

Reply:
Up to you. I find XML hard to manipulate with sed, but if it's just a small local change and not a structural one, you might get away with it. :wink: